.github/workflows/tekton_task_tests.yaml

```diff
@@ -55,14 +82,32 @@ jobs:
           done
 
           kubectl get pods --namespace tekton-pipelines
+
+          # let tekton controllers startup...
+          sleep 10
```

Comment:
Some suggestion :

```
timeout 30  bash -c 'while ! kubectl get pod -n tekton-pipelines -l "app=tekton-pipelines-controller"  --field-selector=status.phase==Running |grep tekton; do echo "Waiting for tekton..."; sleep 1; done '
```
It seems too much but it has the advantage of waiting the proper time tekton would take to bring up - up to 30 seconds. Useful if for any reason the load is too high and it would take longer than expected 10 seconds, but not more than 30 (or maybe more).

Reply:
by this time, Tekton is running but the feature flags configMap may not have been created. I can add a comment about that.